dss/api/search.py

```diff
@@ -96,31 +96,27 @@ def _es_search_page(es_query: dict,
     if output_format != 'raw':
         es_query['_source'] = False
 
-    # The time for a scroll search context to stay open per page. A page of results must be retreived before this
-    # timeout expires. Subsequent calls to search will refresh the scroll timeout. For more details on time format see:
-    # https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#time-units
-    scroll = '2m'  # set a timeout of 2min to keep the search context alive. This is reset
-
-    # From: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-scroll.html
-    # Scroll requests have optimizations that make them faster when the sort order is _doc. If you want to iterate over
-    # all documents regardless of the order, this is the most efficient option:
-    # {
-    #   "sort": [
-    #     "_doc"
-    #   ]
-    # }
-    sort = {"sort": ["_doc"]}
+    # https://www.elastic.co/guide/en/elasticsearch/reference/5.5/search-request-search-after.html
+    sort = [
+        # "manifest.version:desc",
+        "_uid:desc"]
```

Comment:
Why can't we just sort by bundle UUID and version (FQID)? Then we won't even need to use the opaque `_uid` field which is probably yet another UUID and looks just like a bundle UUID which could be confusing.

I'd prefer the sorting to be meaningful rather than random.